data/coalition/coalition ships.txt

```diff
@@ -695,6 +1150,175 @@ ship "Heliarch Punisher" "Heliarch Punisher (Scrappy)"
 		"Hyperdrive"
 
 
+ship "Heliarch Pursuer"
+	sprite "ship/heliarch pursuer"
+	thumbnail "thumbnail/heliarch pursuer"
+	attributes
+		category "Fighter"
+		licenses
+			Heliarch
+		"cost" 1029000
+		"shields" 2200
+		"hull" 1600
+		"required crew" 1
+		"bunks" 1
+		"mass" 31
+		"drag" 0.8
+		"heat dissipation" 0.99
+		"outfit space" 105
+		"weapon capacity" 19
+		"engine capacity" 36
+		weapon
+			"blast radius" 15
+			"shield damage" 150
+			"hull damage" 200
+			"hit force" 350
+	outfits
+		"Ion Rain Gun"
+		
+		"Large Collector Module"
+		"Small Collector Module"
+		"Small Battery Module" 3
+		"Small Shield Module"
+		"Small Repair Module"
+		
+		"Small Steering Module" 2
+		"Small Thrust Module" 2
+	
+	engine 0 34.5
+	engine 7 33.5
+	engine -7 33.5
+	gun 0 -35.5
+	explode "tiny explosion" 20
+	explode "small explosion" 10
+	description `Occasionally, when a Heliarch cadet shows promising results in dogfighting simulations, they are allowed to fly a ship of their own, that being the Pursuer.`
```

Comment:
```suggestion
	description `Occasionally, when a Heliarch cadet shows promising results in dogfighting simulations, they are allowed to fly a ship of their own: the Pursuer.`
```
Iffy about the grammatical correctness of the sentence as-is. Could be right; I just don't know. I tried to suggest something that I know is right and still kept the rhythm of your original sentence.